tests/difference/core/model/src/properties.ts

```diff
@@ -182,6 +182,65 @@ function stakingWithoutSlashing(hist: BlockHistory): boolean {
   return true;
 }
 
+/**
+ * Checks the validator set replication property as defined
+ * https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/system_model_and_properties.md#system-properties
+ *
+ * @param hist A history of blocks.
+ * @returns Is the property satisfied?
+ */
+function validatorSetReplication(hist: BlockHistory): boolean {
+  // return true;
+  const blocks = hist.blocks;
+  let good = true;
+  blocks[C].forEach((b: CommittedBlock, hC: number) => {
+    if (hC < 1) {
+      // The model starts at consumer height 0, so there is
+      // no committed block at height - 1. This means it does
+      // not make sense to try to check the property for height 0.
+      return
+    }
+    const ss = b.invariantSnapshot;
+    // Get the vscid of the last update which dictated
+    // the consumer valset valsetC committed at hC-1
+    const vscid = ss.hToVscID[hC];
+    // The VSU packet was sent at height hP-1
+    const hP = ss.vscIDtoH[vscid];
+    // Compare the validator sets at hC-1 and hP-1
```

Comment:
Explain better the logic here. You can refer to the spec on how `ss.hToVscID` and `ss.vscIDtoH` are set.

Reply:
The function is pretty extensively documented in the latest commit with link to the spec

https://github.com/cosmos/interchain-security/blob/danwt/diff-test-core-validator-set-replication/tests/difference/core/model/src/properties.ts#L185-L192

I think I had not pushed the last commits when you reviewed